syntax = "proto3";
import "google/protobuf/timestamp.proto";


option objc_class_prefix = "ETG";
option java_multiple_files = true;
option java_package = "etg.data.entry";
option java_outer_classname = "EtgProto";
option csharp_namespace = "Etg.Data.Entry";

package etg.data.entry;

message GetEntryStatusRequest {
    string entry_id = 1;
}

message GetEntryStatusResponse {
    string entry_id = 1;
    string status_text = 2;
    google.protobuf.Timestamp declare_date = 3;
}

service EntryDataService {
    rpc GetEntryStatus(stream GetEntryStatusRequest) returns (stream GetEntryStatusResponse) {}
}